src/main/java/Events.java

```diff
@@ -0,0 +1,22 @@
+public class Events extends Task {
+
+    protected String MeetingTime;
+
+    public Events(String description, String MeetingTime) {
+        super(description);
+        this.MeetingTime = MeetingTime;
+    }
+
+    public void setMeetingTime(String MeetingTime) {
+        this.MeetingTime = MeetingTime;
+    }
```

Comment:
Can add comments to explain what each function is doing 
Eg: to set the meeting time of the Events